charts/karpenter-crd/values.yaml

```diff
@@ -0,0 +1,7 @@
+webhook:
+  # -- Whether to enable the webhooks and webhook permissions.
+  enabled: true
+  serviceName: karpenter
+  serviceNamespace: kube-system
```

Comment:
Isn't this just the release namespace? Is there a need to make the namespace configurable?

Reply:
These are the default values, right? So this is where someone would change the value to whatever they want.

Reply:
I was thinking that user could install the karpenter-crd in a different namespace then the karpenter deployment

Reply:
Probably not likely, but it probably doesn't matter a ton. I'm fine with it as-is.